app/strings.php

```diff
@@ -0,0 +1,1365 @@
+    '2.0 Projects' => [
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2478%3A64920
+        'Projects/Individual/Participating' => [
+            __('Notifications'),
+            __('You have been invited to participate in :organization project, :project. They would like you to join for the :engagement engagement.'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2478%3A65288
+        'Projects/Browse all projects' => [
+            __('Projects'),
+            __('Browse all projects'),
+            __('Search'),
+            __('Search...'),
+            __('Status'),
+            __('Upcoming'),
+            __('Cancelled'),
+            __('Who they’re seeking'),
+            __('Seeking Consultation Participants'),
+            __('Seeking Accessibility Consultants'),
+            __('Seeking Community Connectors'),
+            __('Initiated by'),
+            __('Community organization'),
+            __('Regulated organization'),
+            __('Disability and Deaf groups they are looking for'),
+            __('Vision including blind, partially sighted'),
+            __('Deaf people'),
+            __('Hard-of-hearing'),
+            __('Physical and mobility'),
+            __('Pain'),
+            __('Communication'),
+            __('Neurodivergence'),
+            __('Developmental'),
+            __('Learning'),
+            __('Cognitive'),
+            __('Mental health'),
+            __('Multiple disabilities'),
+            __('Episodic and invisible disabilities'),
+            __('DeafBlind'),
+            __('Body differences including size, limb and facial differences'),
+            __('Temporary disabilities'),
+            __('Other disability'),
+            __('Supporters'),
+            __('Virtual or in-person'),
+            __('Virtual - video chat'),
+            __('Virtual - phone call'),
+            __('In-person'),
+            __('Service location'),
+            __('Payment method'),
+            __('Paid'),
+            __('Volunteer'),
+            __('Language'),
+            __('Sectors'),
+            __('Areas of impact'),
+            __('Recruitment method'),
+            __('Open call'),
+            __('Matching service'),
+            __('Community Connector'),
+            __('Previous engagement'),
+            __('Apply filter'),
+            __('Project by :organization'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A5877
+        'create new project - 3' => [
+            __('Steps for creating your project'),
+            __('Not started'),
+            __('Preview'),
+            __('Publish project'),
+            __('Once you publish your project, others can find it.'),
+            __('French translation of project goals'),
```

Comment:
I think all of these types of strings need to be replaced with `:language translation of project goals`, as we can interpolate the language in a consistent manner